features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql

```diff
@@ -4,6 +4,12 @@ CREATE TABLE IF NOT EXISTS DM_DEVICE_TYPE (
      PRIMARY KEY (ID)
 )ENGINE = InnoDB;
 
+CREATE TABLE IF NOT EXISTS DM_DEVICE_CERTIFICATE (
+      ID                    INTEGER auto_increment NOT NULL,
```

Comment:
Fix formatting.

Reply:
Fixed in https://github.com/wso2/carbon-device-mgt/commit/c161d7b65941c2afd6e24b80ba64f641d75f5351